ui/src/app/pages/user-disabled.tsx

```diff
@@ -0,0 +1,57 @@
+import {Component} from '@angular/core';
+import * as React from 'react';
+
+import {StyledAnchorTag} from 'app/components/buttons';
+import {FlexColumn} from 'app/components/flex';
+import {Header} from 'app/components/headers';
+import {backgroundStyleTemplate, SIGNED_OUT_HEADER_IMAGE, SignInStep, StepToImageConfig} from 'app/pages/login/sign-in';
```

Comment:
I wasn't sure where to put this page. It is designed to look like the logged out pages, but I don't really like importing from different modules like this. Any ideas? (I also ended up copying some other code, that could probably be abstracted some more?

Reply:
It could be abstracted some more, but it's not too bad as-is. The most fragile part looks like the backgroundStyleTemplate method call... I don't think including the bg image is critical to the disabled page, so I might be inclined to leave that out in the name of maintainability, but keep the header image (which could more easily be pulled out into a separate constants file, but also isn't a critical to-do IMO).

Reply:
The background image is actually a relatively clear part of Lou's mock/Karthik's design. I think they wanted it to look like the signin page, so that it was a similar page for users who sign in and find themself disabled:
https://projects.invisionapp.com/d/main#/console/12820961/408493657/inspect